crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -1,43 +1,40 @@
+mod command_queue;
+
 use crate::{
     bundle::Bundle,
     component::Component,
     entity::{Entities, Entity},
     world::World,
 };
 use bevy_utils::tracing::debug;
+use command_queue::CommandQueueInner;
 use std::marker::PhantomData;
 
 /// A [`World`] mutation.
 pub trait Command: Send + Sync + 'static {
-    fn write(self: Box<Self>, world: &mut World);
+    fn write(self, world: &mut World);
```

Comment:
We should document (in a non-doc-comment) that if Send ever gets removed from these, then CommandQueue's Send impl is no longer true.